stories/checkboxes.stories.js

```diff
@@ -70,6 +91,29 @@ storiesOf( 'Global', module ).add(
 					Disabled Checkbox
 				</Checkbox>
 			</div>
+
+			<div>
```

Comment:
Updated the existing Checkbox story for parity with the GM3 Checkbox story, to help comparing old to new.